test/screenshot/spec/mdc-button/fixture.scss

```diff
@@ -54,7 +54,7 @@ $custom-button-custom-horizontal-padding: 24px;
 }
 
 .custom-button--corner-radius {
-  @include mdc-button-corner-radius($custom-button-custom-corner-radius);
+  @include mdc-button-shape-radius($custom-button-custom-corner-radius);
```

Comment:
Should we also update the class name and the name of the button and card mixin test pages since we're updating the name of the mixin?

This also needs to be done for chips.